docs/course/index.md

```diff
@@ -493,3 +485,35 @@ While every course should use the different roles in a way that best suits the c
 | TAs involved in grading              |         None          | Student data editor |
 | Other TAs                            |         None          | Student data viewer |
 | Instructors from other classes       | Course content viewer |        None         |
+
+## Course-wide options
+
+These options, which apply to all instances of your course, can be set in the `infoCourse.json` file. The properties should be set within an object named `options` as shown in the example above.
+
+| Property                 | Default | Description                                                                                                                                                                                                                                                 |
+| ------------------------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `useNewQuestionRenderer` | `false` | This controls whether to use the "legacy" renderer or the "new" renderer. Although the default is `false` currently to avoid issues with longstanding courses on the platform, all courses should set this to `true`, especially if they are newly created. |
+
+### Question renderers
+
+The "legacy" renderer is the original renderer used in PrairieLearn. It renders elements alphabetically in order of their name and thus provides inconsistent support for nesting elements. It should be avoided in new courses.
+
+The "new" question renderer processes elements in a tree, which allows it to consistently and predictably support nested elements. It's also more performant. If you have an existing course that predates the new renderer, you should migrate to it at their earliest convenience by setting `"useNewQuestionRenderer": true` in your `infoCourse.json` file:
```

Comment:
`at their earliest` sounds a little funny. "your earliest"?

Reply:
Good catch, that was left over from an intermediate wording of all this.

```suggestion
The "new" question renderer processes elements in a tree, which allows it to consistently and predictably support nested elements. It's also more performant. If you have an existing course that predates the new renderer, you should migrate to it at your earliest convenience by setting `"useNewQuestionRenderer": true` in your `infoCourse.json` file:
```